gui/projects.php

```diff
@@ -26,13 +32,33 @@ function projects_list(){/*{{{*/
 	}
 	echo "</table>";
 
+	//new project
 	echo "<br><br><br>
 	<form method=POST>
 		<input autocomplete=off type=text placeholder='new project name' name=new_project required pattern='\w{1,15}' title='max 15 of alphanumeric characters'> 
 		<input autocomplete=off type=submit name=submit value='add'>
 	</form>
 	";
 
+	//rename project
+	echo "<br>
+	<div>Rename current project (<i>$current_project</i>) as:
+	<form method=POST>
+		<input autocomplete=off type=text placeholder='new project name' name=rename_project required pattern='\w{1,15}' title='max 15 of alphanumeric characters'> 
+		<input autocomplete=off type=submit name=submit value='rename'>
+	</form></div>
+	";
+
+	//rename scenario
+	echo "<br>
+	<div>Rename current scenario (<i>$current_project/$current_scenario</i>) as:
+	<form method=POST>
+		<input autocomplete=off type=text placeholder='new scenario name' name=rename_scenario required pattern='\w{1,15}' title='max 15 of alphanumeric characters'> 
+		<input autocomplete=off type=submit name=submit value='rename'>
+	</form></div>
+	";
+
+	//copy scenario
 	echo "<br>
 	<div>Copy current scenario as:
```

Comment:
Optionally we can alter also this line just to unify input fields labels:
`<div>Copy current scenario (<i>$current_project/$current_scenario</i>) as:`

Reply:
labels unified